native/src/utils/renderHtml.ts

```diff
@@ -118,10 +118,16 @@ const renderHtml = (
       src: ${getFontFaceSource('Raleway-Bold')};
     }
     @font-face {
-      font-family: 'Lateef';
+      font-family: 'Noto Sans Arabic';
       font-style: normal;
       font-weight: 400;
-      src: ${getFontFaceSource('Lateef')};
+      src: ${getFontFaceSource('NotoSansArabic-Regular')};
+    }
+    @font-face {
+      font-family: 'Noto Sans Arabic';
+      font-style: normal;
+      font-weight: 400;
```

Comment:
I guess:
```suggestion
      font-weight: 700;
```

Reply:
first one is regular 400 weight, second one is bold with 700. For me everything should be fine